index.html

```diff
@@ -79,271 +80,240 @@ <h2 id="connect">LET'S CONNECT</h2>
               </li>
 
-              <li class="year">2015</li>
+              <li class="year"> </li>
             </ul>
             <p class="project_description">
               A daily selection of privately personalized reads; no accounts or
               sign-ups required.
             </p>
             <ul class="code_language">
-              <li class="ht1"><a class="html" href="#">html</a></li>
+              <li class="ht1"><a class="html" href="#"> </a></li>
               <li class="ht2"><a class="ruby" href="#">Ruby on rails</a></li>
-              <li class="ht3"><a class="css" href="#">css</a></li>
-              <li class="ht4"><a class="javascript" href="#">javascript</a></li>
+              <li class="ht3"><a class="css" href="#"> </a></li>
+              <li class="ht4"><a class="javascript" href="#"> </a></li>
             </ul>
-            <div class="link_to_project">
-              <a class="check_Project" href="#">See Project</a>
-            </div>
+            <button class="link_to_project" id="3"> See Project </button>
           </div>
         </div>
 
-        <div class="work works3">
-        <img class="secondimage" src="images/Snapshoot Portfolio (7).png" alt="Project snapshoot" />
+        <div class="works works3">
+ 
           <div class="Project2">
-            <h3 class="project_name">Uber Navigation</h3>
+            <h3 class="project_name"> </h3>
             <ul class="client">
-                <li id="h3">Uber</li>
+                <li id="h3"> </li>
               <li class="Counter">
                 <img src="images/Counter.png" alt="" />
               </li>
 
-              <li class="langue">Lead Developer</li>
+              <li class="langue"> </li>
               <li class="Counter">
                 <img src="images/Counter.png" alt="" />
               </li>
 
-              <li class="year">2018</li>
+              <li class="year"> </li>
             </ul>
             <p class="project_description">
               A daily selection of privately personalized reads; no accounts or
               sign-ups required.
             </p>
             <ul class="code_language">
-              <li class="ht1"><a class="html" href="#">html</a></li>
+              <li class="ht1"><a class="html" href="#"> </a></li>
               <li class="ht2"><a class="ruby" href="#">Ruby on rails</a></li>
-              <li class="ht3"><a class="css" href="#">css</a></li>
-              <li class="ht4"><a class="javascript" href="#">javascript</a></li>
+              <li class="ht3"><a class="css" href="#"> </a></li>
+              <li class="ht4"><a class="javascript" href="#"> </a></li>
             </ul>
-            <div class="link_to_project">
-              <a class="check_Project" href="#">See Project</a>
-            </div>
+            <button class="link_to_project" id="4"> See Project </button>
           </div>
         </div>
-      </div>
-     
-
-
-
+      </div>f 
```

Comment:
- [OPTIONAL] Since the previous code reviewer forgot to mention this, I have to put it as an 'OPTIONAL' requirement. But all of the HTML code related to the projects section should be removed and refactored so you create this section dynamically with JavaScript. I see that part of the content is created dynamically, but the hard-coded HTML should be removed. Kindly be aware of this and fix this for the next submission.
 - 
<img width="999" alt="image" src="https://user-images.githubusercontent.com/61919617/211044895-b6a277df-99fc-4b9c-ae11-a2cba1555418.png">

 - [ ] Your modal is almost perfect for the project, but unfortunately, it does not pass on the requirements. The modal should not be transparent, and on the desktop version, it should be displayed in the center of the screen. Kindly fix these issues to make a better user experience and looks more similar to the Figma template.
 - Your design:
<img width="1216" alt="image" src="https://user-images.githubusercontent.com/61919617/211045994-573bc213-6842-43fa-937d-4066b57f2d93.png">